Phoron tank fix

Flaksim opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Flaksim committed:
Phoron canister fix

Replaced all phoron canisters with the engine fix variant.

Flaksim committed:
outline tile fix

fixes outlines

Flaksim committed:
Merge pull request #3 from HaloSpaceStation/alpha

update fork

Flaksim committed:
Corvette red alert

Adds an alarm button to the corvette bridge.

Flaksim committed:
Fix on outpost

fixes a misplaced boot.

Flaksim committed:
Unngoy equipment fix

Fixes unggoy spawning without equipment

Flaksim committed:
Ugnnoy methane nest

gives uggnoy area methane

Flaksim committed:
valve fix

covie ship valve fix

Flaksim committed:
fixes space area on exo

Flaksim committed:
Map fixes

Various map fixes, biofoam fix attempt, STILL doesnt work.

X0-11 commented on maps/first_contact/maps/UNSC_Corvette.dmm:
```diff
@@ -95,115 +95,113 @@
+"cr" = (/obj/structure/cable/blue{icon_state = "1-2"},/turf/simulated/floor/tiled,/area/om_ships/unscpatrol)
+"cs" = (/obj/machinery/light,/obj/structure/table/standard,/obj/item/weapon/paper_bin,/obj/item/weapon/pen,/turf/simulated/floor/tiled,/area/om_ships/unscpatrol)
+"ct" = (/obj/structure/bed/chair{tag = "icon-chair_preview (NORTH)"; icon_state = "chair_preview"; dir = 1},/turf/simulated/floor/tiled,/area/om_ships/unscpatrol)
+"cu" = (/obj/structure/cable/blue{icon_state = "1-2"},/obj/machinery/button/toggle/alarm_button/corvette/v4{pixel_x = 32},/turf/simulated/floor/tiled,/area/om_ships/unscpatrol)
```
The alarm button should be randomly-chosen via /obj/random_corvette_alarm rather than preset.

Flaksim replied:
I put the  correct one in.

Flaksim committed:
fixes

Food and crowbars added to kig-yar ship.
Biofoam fixed
Extra docking umbilicals added to planets.
Methane den finished.

Flaksim committed:
Cleaning supplies

Added matter bins and cleaning supplies to most areas.

Flaksim committed:
New overmap object: UEG Listening post.

*Adds a small 50x50 UEG listening post to the overmap.
*Contains mainly comms equipment (high RP job only)
*Two slots for Listening Post Technicians added.

Flaksim committed:
Spawns increased

Slightly increased URF and grunt slots to counter new UEG jobs.
Fixed space turf at asteroid.
